apps/indexer/test/indexer/fetcher/internal_transaction_test.exs

```diff
@@ -287,6 +287,7 @@ defmodule Indexer.Fetcher.InternalTransactionTest do
 
       assert %{block_hash: block_hash} = Repo.get(PendingBlockOperation, block_hash)
 
+      # assert {:retry, [block.number]} == InternalTransaction.run([block.number, block.number], json_rpc_named_arguments)
```

Comment:
That should still be working, right? Let's uncomment?

Reply:
I changed it back because it went to infinite loop when it failed. It will get the pending blocks from the DB instead (function `Indexer.Fetcher.InternalTransaction.init`)